exalearn-ui/.prettierrc

```diff
@@ -4,12 +4,12 @@
     "endOfLine": "auto",
     "trailingComma": "none",
     "arrowParens": "always",
-    "jsxBracketSameLine": true, 
-    "useTabs": true, 
-    "tabWidth": 4, 
+    "jsxBracketSameLine": true,
+    "useTabs": true,
+    "tabWidth": 2,
```

Comment:
Why did you change it on 2?

Reply:
We've discussed that we have 2 tabWidth, don't we?

Reply:
We have discussed that we have 2 tabwidth, don't we?

Reply:
We made a call with @HrebenYury and decided that 4 is okay, it was in his PR with linter and prittier